web/src/lib/components/faces-page/people-list.svelte

```diff
@@ -1,66 +1,35 @@
 <script lang="ts">
-  import { maximumLengthSearchPeople, timeBeforeShowLoadingSpinner } from '$lib/constants';
-  import { handleError } from '$lib/utils/handle-error';
-  import { searchNameLocal } from '$lib/utils/person';
-  import { searchPerson, type PersonResponseDto } from '@immich/sdk';
+  import { type PersonResponseDto } from '@immich/sdk';
   import { createEventDispatcher } from 'svelte';
   import FaceThumbnail from './face-thumbnail.svelte';
-  import SearchBar from '../elements/search-bar.svelte';
+  import SearchPeople from '$lib/components/faces-page/search-people.svelte';
 
   export let screenHeight: number;
   export let people: PersonResponseDto[];
   export let peopleCopy: PersonResponseDto[];
   export let unselectedPeople: PersonResponseDto[];
 
   let name = '';
-  let searchWord: string;
-  let isSearchingPeople = false;
+  let showPeople: PersonResponseDto[];
```

Comment:
Does this `showPeople` mean search result?

Reply:
Yes, it's for the people shown to the users.